### ISSUE TITLE
Missing brackets in expression for anchor token in Grammar.pp

### ISSUE THREAD

**Issue by turkanis** [hoaproject/Regex#34]
In definition of the token "anchor", the sequence of character "bBAZzG" should occur within a character class:
```
diff --git a/Source/Grammar.pp b/Source/Grammar.pp
index 4176085..15e1714 100644
--- a/Source/Grammar.pp
+++ b/Source/Grammar.pp
@@ -106,7 +106,7 @@
 // Please, see PCRESYNTAX(3), General Category properties, PCRE special category
 // properties and script names for \p{} and \P{}.
 %token character_type            \\([CdDhHNRsSvVwWX]|[pP]{[^}]+})
-%token anchor                    \\(bBAZzG)|\^|\$
+%token anchor                    \\([bBAZzG])|\^|\$
 %token match_point_reset         \\K
 %token literal                   \\.|.
```

**Comment by Hywan** [comment]
Hello and thanks for the bug report.

That's totally correct! Do you want to make a PR?

**Comment by turkanis** [comment]
Thanks. See #35.

**Comment by vonglasow** [comment]
fixed by #35
